ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -98,54 +97,51 @@ const CancelSpeedupPopover = () => {
       <div className="cancel-speedup-popover__wrapper">
         <Text
           alignItems={AlignItems.center}
-          display={DISPLAY.FLEX}
+          display={Display.Flex}
           variant={TextVariant.bodySm}
-          as="h6"
-          marginTop={0}
           marginBottom={2}
+          paddingBottom={2}
+          className="cancel-speedup-popover__description"
         >
           {t('cancelSpeedUpLabel', [
-            <strong key="cancelSpeedupReplace">{t('replace')}</strong>,
+            <strong key="cancelSpeedupCancel">{t('replace')}</strong>,
```

Comment:
Question, why was the key changed on this? Also...why is the key needed? This might be a question for @georgewrmarshall

Reply:
Good question! This may have been a mistake. I'll remove it

Reply:
Ah right I'm not super familiar with locales but because it's in an array it needs a `key`

<img width="806" alt="Screenshot 2023-06-20 at 4 10 17 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/21ae3e03-31d8-4242-9763-6cb19b98c959">